PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/actions/WebDeployAction.java

```diff
@@ -62,50 +63,38 @@ public void actionPerformed(AnActionEvent event) {
             return;
         }
 
-        ApplicationManager.getApplication().invokeLater(() -> {
-            runConfiguration(project);
-        });
+        ApplicationManager.getApplication().invokeLater(() -> runConfiguration(project));
     }
 
     private void runConfiguration(Project project) {
-        final RunManagerImpl manager = RunManagerImpl.getInstanceImpl(project);
```

Comment:
PR here: https://github.com/Microsoft/azure-tools-for-java/pull/762
We want to add a menu item action that can make the user able to open a run configuration with some before run tasks setted(i.e. maven package), which can be used for deploying web app on Azure.

The feature is not finished yet. And why we need to change it is that from 2017.1 to 2017.2, Intellij's implementation of runManager has a big change, especially for the before run tasks management. So we are trying to figure out a way to make the code compatibale for both 2017.1 and 2017.2